### ISSUE TITLE
Place cursor at the end of text fields

### ISSUE THREAD

**Issue by xxssmaoxx** [ueokande/vim-vixen#601]
### I'm opening this issue because:

- [ ] I'll report a bug
- [x] I'll propose a new feature

### Description
When focusing a text field the cursor is always shown at the start of the line, but it's a bad position in my opinion. 
Especially in search engines when you search a little too much specific I usually delete a word or two and it is very easy using ctrl+backspace (it's very unconfortable to do with ctrl+del since the del key is smaller).  Also the opposite case is possible, when you get too many results, and you want to append a couple more keywords. 
So i propose to always focus text fields at the end of the current text.

I would be very happy to have this feature in future and I'm sure many others.

**Comment by LeonT-A** [comment]
I agree, this would be incredibly handy. At the moment I press the 'end' key to jump me to the end of the line. Though I'm sure it would be somewhat faulty every once in a while, having it auto-press the end key when a text field comes into focus would be great.
